python/dgl/graphbolt/impl/neighbor_sampler.py

```diff
@@ -118,9 +118,15 @@ def sample_subgraphs(self, seeds):
         # Enrich seeds with all node types.
         if isinstance(seeds, dict):
             ntypes = list(self.graph.node_type_to_id.keys())
+            device = None
+            dtype = None
+            for _, seed in seeds.items():
```

Comment:
it is not intuitive to use loop here, add comment.

Reply:
added